### ISSUE TITLE
Strange tap/drag behavior on iPad Mini

### ISSUE THREAD

**Issue by johndubya** [leongersen/noUiSlider#378]
We have an iPad Mini in the office and were testing our website on it. All other devices we have tried so far seem to work great with noUiSlider, but the iPad had this strange issue where if you slide your finger across the slider too quickly, it either a) doesn't register anything, or b) moves the slider immediately to where your finger let off without showing the tooltips. I uploaded a video to show an example of what was happening. Any ideas?

https://www.youtube.com/watch?v=LGlqKmBvt4o

**Comment by danbuntu** [comment]
I'm also seeing the same issue.

I have a cordova/ phonegap (which uses web view) app that is using the latest nouislider and jquery. I'm testing on a moto g running cyanogenmod 12.

The sliders are very laggy to start. If you press on them for a second they pop into life.

The demos and the web version of my app run fine on the same pone using google chrome.

**Comment by leongersen** [comment]
How do you propose to debug this?

**Comment by danbuntu** [comment]
I can knock up an apk file if you want with a slider in it. I can then check that the issues exists without the other various scripts and libraries that my app includes. That will then demonstrate the issue and allow some debugging information collection via chrome remote inspection.

If you're then unable to find anything obvious when looking at the apk then we can perhaps raise this with the google android team and see if they have any thoughts.

**Comment by danbuntu** [comment]
I had a play with these over the weekend and built a simple cordova project with just the slider and it worked perfectly.
I then started to dig more into the code on my project and tracked the issue I was having down to the css themeing I was using. I had the sliders styles to use a round button that grew when it was selected and a thin line for the actual slider part.
When I reverted that back to the standard css this started working fine.
